support/Procfile.linux

```diff
@@ -0,0 +1,8 @@
+redis: redis-server
```

Comment:
Not a blocker, but I'm curious why the linux Procfile starts redis and the mac one doesn't.